src/js/helpers/apiRequest.js

```diff
@@ -6,12 +6,21 @@
 import Axios, { CancelToken } from 'axios';
 import kGlobalConstants from 'GlobalConstants';
 
+const mockUrl = `http://localhost:5000/api/`;
+const localUrl = `http://localhost:8000/api/`;
+
+const getBaseUrl = (params) => {
+    if (params.isMocked) return mockUrl;
```

Comment:
Would we want to change these to what they are doing?

useMockBaseURL
useLocalBaseURL